web/tests/copy_and_paste.test.js

```diff
@@ -2,11 +2,55 @@
 
 const {strict: assert} = require("assert");
 
-const {zrequire} = require("./lib/namespace");
+const {zrequire, mock_esm} = require("./lib/namespace");
 const {run_test} = require("./lib/test");
 
 const copy_and_paste = zrequire("copy_and_paste");
 
+mock_esm("../src/stream_data", {
+    slug_to_name() {
+        return "Rome";
+    },
+});
```

Comment:
Don't do a mock, just initialize the `stream_data` module with a single stream object if that's what you need.

Reply:
Done